helm/cloud-provider-cloud-director-app/charts/cloud-provider-for-cloud-director/values.yaml

```diff
@@ -0,0 +1,38 @@
+basicAuthSecret:

```

Comment:
is defined in the top level chart, I would not repeat but use instead `global` var

Reply:
This is to make the chart as generic as possible with default values to make it usable independently of the top level chart.

Reply:
my background as dev warn me when I see double code :D